Include/pystate.h

```diff
@@ -142,9 +145,6 @@ typedef struct _ts {
     PyObject *coroutine_wrapper;
     int in_coroutine_wrapper;
 
-    Py_ssize_t co_extra_user_count;
-    freefunc co_extra_freefuncs[MAX_CO_EXTRA_USERS];
```

Comment:
`PyThreadState` is public, and you're changing the size of the struct, so this *is* a problematic ABI break. Keeping these fields the same size, but changing their names to `_preserve_36_ABI_1` and `_preserve_36_ABI_2` should address most of that concern.